service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/AdditionalApplicationsUploadedEventHandler.java

```diff
@@ -123,14 +124,33 @@ public void notifyApplicant(final AdditionalApplicationsUploadedEvent event) {
             final CaseData caseData = event.getCaseData();
             final OrderApplicant applicant = event.getApplicant();
 
-            if (applicant.getType() == ApplicantType.LOCAL_AUTHORITY) {
-                Collection<String> emails = inboxLookupService.getRecipients(
-                    LocalAuthorityInboxRecipientsRequest.builder().caseData(caseData).build());
-                sendNotification(caseData, emails);
+            if (applicant.getType() == LOCAL_AUTHORITY) {
+
+                final RecipientsRequest recipientsRequest = RecipientsRequest.builder()
+                    .caseData(caseData)
+                    .secondaryLocalAuthorityExcluded(true)
+                    .build();
+
+                final Collection<String> recipients = localAuthorityRecipients.getRecipients(recipientsRequest);
+
+                sendNotification(caseData, recipients);
```

Comment:
vv minor but would it be nicer to have one sendNotification call after the if block and the contents if each if block just adds to the collection?